packages/website/src/components/FinancialContributors/Sponsors/styles.module.css

```diff
@@ -19,6 +19,7 @@
 }
 
 .sponsorsTier img {
+  background: white;
```

Comment:
Tangentially related: I noticed sponsor logos such as Nrwl weren't showing up in dark mode. Branding!